npp.py

```diff
@@ -12,26 +12,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-## ------- Set up our list of numbers -------
-S = [25, 7, 13, 31, 42, 17, 21, 10]
-
-## ------- Set up our QUBO dictionary -------
+## ------- import packages -------
+from dwave.system import DWaveSampler, EmbeddingComposite
 
 # TODO:  Add code here to define your QUBO dictionary
+def define_QUBO(S):
 
-## ------- Run our QUBO on the QPU -------
-# TODO:  Choose QPU parameters
-chainstrength = 1
-numruns = 10
+    Q = {}
 
-# Run the QUBO on the solver from your config file
-from dwave.system import DWaveSampler, EmbeddingComposite
+    # Add QUBO construction here
+    
+    return Q
+
+# TODO:  Choose QPU parameters in the following function
+def run_on_QPU(Q, sampler):
```

Comment:
ditto on docstring and on snake-case

```
def run_on_qpu(Q, sampler):
    ```What this function does.

    Args:
        Q(dict): a representation of a QUBO
        sampler(dimod.Sampler): a sampler that uses the QPU
    ```
```